test/libsolidity/syntaxTests/tupleAssignments/nowarn_explicit_singleton_token_expression.sol

```diff
@@ -0,0 +1,8 @@
+contract C {
+	function f() public pure {
+        uint a;
+		(a,) = (uint(1),);
+	}
+}
+// ----
+// Warning: (59-76): Different number of components on the left hand side (2) than on the right hand side (1).
```

Comment:
Why is the RHS accepted, but being told it is 1 component?

Reply:
This is the way to distinguish between a parenthesized expression and a tuple with one component: https://solidity.readthedocs.io/en/v0.4.23/control-structures.html#destructuring-assignments-and-returning-multiple-values (in the code)